src/containers/forms/LogActivityForm.jsx

```diff
@@ -51,15 +51,15 @@ class LogActivityForm extends Component {
       const {
         description,
         category,
-        date,
+        activityDate,
         numberOf,
         activityTypeId,
       } = selectedItem;
 
       const formTitle = 'Edit Activity Request Form';
       const btnText = 'Update';
       return {
-        date,
+        date: activityDate,
```

Comment:
@puritybirir Can we return the `activityDate` instead of assigning it to the `date` key?

Reply:
I have refactored the files to reflect the same